knowledge-base/using-ms-access-database-in-the-report .md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using MS Access Database in the Report 
+description: Since version 13.1.19.618 we provide support for MS Access. 
+type: how-to
+page_title: How to use MS Access Database in Telerik Report?
+slug: using-ms-access-database-in-the-report 
+position: 
+tags: 
+ticketid: 1417017
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>13.1.19.618</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Telerik Reporting</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+This KB article lists the required steps for setting MS Access Database and using it in the report.
+
+## Solution
+1. Create a sample database in **Access**:
+![](resources/AccessDB.png)
```

Comment:
there is no resource called AccessDB.png in the resource folder. Please add it :)